addons/pause/addon.json

```diff
@@ -25,7 +25,7 @@
       "matches": ["https://scratch.mit.edu/projects/*"]
     }
   ],
-  "tags": ["editor", "beta"],
+  "tags": ["editor"],
```

Comment:
Recommended?

Reply:
Scratch Addons only note: let's also mark this addon as new and feature it in the release.